reactor-core/src/test/java/reactor/core/scheduler/ElasticSchedulerTest.java

```diff
@@ -276,4 +280,62 @@ else if (i == fastCount - 1) {
 			LOGGER.info("{} threads active post shutdown", Thread.activeCount() - otherThreads);
 		}
 	}
+
+	@Test
+	public void doesntRecycleWhileRunningAfterDisposed() throws Exception {
+		Scheduler s = Schedulers.newElastic("test-recycle");
+		((ElasticScheduler)s).evictor.shutdownNow();
+
+		try{
+			AtomicBoolean stop = new AtomicBoolean(false);
+			Disposable d = s.schedule(() -> {
+				// simulate uninterruptible computation
+				for (;;) {
+					if (stop.get()) {
+						break;
+					}
+				}
+			});
+			Thread.sleep(100);
```

Comment:
since it is not time-sensitive, please add a `CountDownLatch` to detect the start of the submitted `Runnable` instead of waiting for a fixed amount of time